deployment/docker/traffic-replayer/run_app.sh

```diff
@@ -10,22 +10,22 @@ do
   # Verify connection with TC can be made before disconnecting via 1 second timeout and moving forward
   while ! nc -v -w 1 localhost 9220
   do
-    echo 'Attempting to connect to Traffic Comparator...'
+    >&2 echo 'Attempting to connect to Traffic Comparator...'
     sleep 2
   done
 
   # Allow time for TC to begin listening after terminated connection
-  sleep 10
+  sleep 7
 
   touch triples.log
 
   # These commands look a bit backwards but ultimately will receive input from CW-Puller via netcat and then
   # process with the Replayer jar before ultimately printing the produced triples to stdout via tee and sending to TC via netcat
   # Note: Unbuffer has not been verified to actually be effective here. stdbuf was ruled out for its caveat with tee (https://linux.die.net/man/1/stdbuf)
   unbuffer tail -f triples.log | tee /dev/stderr | nc -v localhost 9220 &
-  nc -v -l -p 9210 | java -jar trafficReplayer-uber.jar -o triples.log "$target_endpoint"
+  nc -v -l -p 9210 | java -jar trafficReplayer-uber.jar "$target_endpoint" -o triples.log --insecure
```

Comment:
are you unbuffering the output to triples.log from the java process?

Reply:
What happens if the uberjar has multiple mains() defined within different source jars?

Reply:
We were using unbuffer as an alternative to stdbuf as it didn't work with tee commands. That said I don't know if we ever confirmed it was useful here and could possibly be removed. 

I've not ran into that issue, but goal is to move away from uberjar here.